clang/docs/SourceBasedCodeCoverage.rst

```diff
@@ -484,10 +484,31 @@ MC/DC Instrumentation
 ---------------------
 
 When instrumenting for Modified Condition/Decision Coverage (MC/DC) using the
-clang option ``-fcoverage-mcdc``, users are limited to at most **six** leaf-level
-conditions in a boolean expression.  A warning will be generated for boolean
-expressions that contain more than six, and they will not be instrumented for
-MC/DC.
+clang option ``-fcoverage-mcdc``, there are two hard limits.
```

Comment:
Suggest simplifying the phrasing here.

> By default, Modified Condition/Decision Coverage (MC/DC) instrumentation is limited to 32767 terms. Users may provide their own limit on the number of terms, `n` by passing ``-Xclang -fmcdc-max-conditions=n``.

Reply:
Do you suggest replacing "there are two hard limits" with the explanation of `max-conditions=32767`?

Reply:
I think it's fine actually.